src/System.Management.Automation/FormatAndOutput/common/DisplayDatabase/typeDataXmlLoader_Table.cs

```diff
@@ -374,12 +373,6 @@ private TableRowDefinition LoadRowEntryDefinition(XmlNode rowEntryNode, int inde
                     }
                     else if (MatchNodeName(n, XmlTags.TableColumnItemsNode))
                     {
-                        if (columnEntriesNodeFound)
-                        {
-                            this.ProcessDuplicateNode(n);
-                            return null; // fatal
-                        }
-
```

Comment:
If compare with multiLineFound the code should follow `columnEntriesNodeFound = true;`. 
If so it is a bug and we should revert the change and fix in new PR. Otherwise columnEntriesNodeFound should be removed at all.

Reply:
The code is unreachable as `columnEntriesNodeFound` is always false, therefore it would seem this is a bug.

I'm not familar with this code but I'm guessing we want something like this in new PR?

```csharp
if (columnEntriesNodeFound)
{
    this.ProcessDuplicateNode(n);
    return null; // fatal
}
columnEntriesNodeFound = true;
```

Reply:
Yes, please revert any bugfixes/semantic changes in this PR. Since this one is quite large, it would be better to keep it focused on the changes described in the title. How to implement bugfixes can be discussed in the new PR

Reply:
Issue opened: #13246